pkg/instance/repository.go

```diff
@@ -12,22 +14,34 @@ import (
 type Repository interface {
 	Link(firstInstance, secondInstance *model.Instance) error
 	Unlink(instance *model.Instance) error
-	Create(instance *model.Instance) error
 	Save(instance *model.Instance) error
 	FindWithParametersById(id uint) (*model.Instance, error)
 	FindByNameAndGroup(instance string, group string) (*model.Instance, error)
 	SaveDeployLog(instance *model.Instance, log string) error
 	FindById(id uint) (*model.Instance, error)
 	Delete(id uint) error
 	FindByGroupNames(names []string) ([]*model.Instance, error)
+	FindWithDecryptedParametersById(id uint) (*model.Instance, error)
```

Comment:
maybe name it `FindByIdWithDecryptedParameters` instead and group it with the `FindById`